backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/query/impl/ElasticsearchSearchQueryImpl.java

```diff
@@ -222,6 +234,7 @@ private Integer defaultedLimit(Integer limit, Integer offset) {
 	}
 
 	private JsonObject doExplain(ElasticsearchSearchIndexContext index, Object id) {
+		timeoutManager.start();
```

Comment:
I just noticed this... you're not supposed to use the timeout manager during `explain()` (cf. documentation). At least we made sure not to use it in the Lucene backend.

Reply:
I agree, I'm removing it...